tests/system/memory_leak/config.sh

```diff
@@ -2,4 +2,9 @@ FLEDGE_URL="http://localhost:8081/fledge"
 TEST_RUN_TIME=3600
 PI_IP="localhost"
 PI_USER="Administrator"
-PI_PASSWORD="password"
\ No newline at end of file
+PI_PASSWORD="password"
+READINGSRATE="100"   # It is the readings rate per second per service
+PURGE_INTERVAL_SECONDS="180"
+STORAGE='sqlite' # postgres, sqlite-in-memory, sqlitelb
+READING_PLUGIN_DB='Use main plugin'
+MEMORY_THRESHOLD=20    # Memory Threshold in percentage
```

Comment:
Add more details in inline comment, What does this mean? I assume we care for `Definitely Lost` only and in any of the component under test.